### ISSUE TITLE
[Bug]: Created spells are empty of data

### ISSUE THREAD

**Issue by Sweed-Le-Deedly** [MinecraftschurliMods/Ars-Magica-Legacy#305]
### Describe the bug

Whenever I try to make a spell of any kind, the resulting book is empty of any data, making it impossible to create spells in the altar.

This could be some kind of incompatibility caused by one of my mods changing the nature of player-written books, but I don't remember if any of my mods have that kind of feature to it and I have about near 86 other mods, so it's hard to tell at the moment.

### Steps to Reproduce

1. Using the Inscription table, place the desired component into the bottom-most slot.
2. Place the desired shape of the spell into the first shape group.
3. Put a book and quill into the central slot, and then take it out. From this, it auto-writes into a signed book, named the same as what was wanted for the spell
4. Read the newely created book, finding it to be one page of no text.
5. Put the spell into the lectern of the crafting altar and toss a blank rune into it.
6. A ding then sounds, indicating that the altar is read to making a spell, but nothing else happens. No gliph trails, no needed items above the lectern.

### Expected behavior

The resulting book should have multiple pages to it, detailing the shapes and components comprising the spell as well as a list of the materials needed to create it.

### Screenshots

![Screenshot 2022-06-05 093756](https://user-images.githubusercontent.com/69220229/172053717-b0b77541-a37f-4c9b-9596-2c87d0322e79.png)
WIth this spell, I want to make a simple fire projectile.

![Screenshot 2022-06-05 093914](https://user-images.githubusercontent.com/69220229/172053753-b6bf4390-d2f6-4b75-bb7e-692b316f0d9f.png)
After dragging the book and quill out of its slot, it auto-writes into a signed book. Checking other mod guides, it should be editable beforehand.

![Screenshot 2022-06-05 093948](https://user-images.githubusercontent.com/69220229/172053794-9d3f5021-2a4a-4867-98dc-529a0bb46d0a.png)
Reading the resulting book, it shows that it's completely empty of any text for the mod to read. This is reguardless of components and shapes used.

### Version

1.18.2-1.1.0

### Forge Version

40.1.25

### Other Mods

- AdChimneys-1.18.2-8.2.0.0-build.0935
- AdPother-1.18.2-6.0.2.0-build.0935
- AdvancementPlaques-1.18.2-1.4.3
- appleskin-forge-mc1.18-2.4.0
- architectury-4.4.68
- backpacked-2.1.6-1.18.2
- balm-3.0.3+0
- betterflight-forge-1.18-1.0.0
- BiomesOPlenty-1.18.2-16.0.0.115
- bon-1.18.2-1.6.2
- campfirespawnandtweaks_1.18.2-1.6
- carryon-1.18.2-1.17.0.8
- catalogue-1.6.2-1.18.2
- ChiselsBits-forge-1.18.2-1.2.90-universal
- cloth-config-6.2.62-forge
- collective-1.18.2-4.25
- configured-1.5.3-1.18.2
- corail_recycler-5.5.1-1.18.2
- corail_woodcutter-1.18.2-2.3.4
- corpse-1.18.2-1.0.1
- Craftable+Horse+Armour++Saddle-1.18-1.8-Forge
- createaddition-1.18.2-20220517a
- createdeco-1.2.7-1.18.2
- create-mc1.18.2_v0.4.1
- create-stuff-additions1.18.2_v1.1.6
- cryingghasts_1.18.2-1.5
- cryingportals_1.18.2-1.2
- CTM-1.18.2-1.1.4+4
- curios-forge-1.18.2-5.0.7.0
- despawningeggshatch_1.18.2-2.4
- dungeons_enhanced-1.18.2-3.1
- dynamiclightsreforged-1.18.2_v1.3.2
- dynamic-music-1.4.1
- effective_fg-1.1
- enchanting_overhauled-1.0.7
- Entity_Collision_FPS_Fix-forge-1.18.2-1.0.0
- flywheel-forge-1.18-0.6.2
- foodeffects-1.18.2-1.1.3
- ForgeEndertech-1.18.2-9.0.2.0-build.0935
- geckolib-1.18-forge-3.0.18
- glowinc-1.18.2_1.2.3
- healingcampfire_1.18.2-3.4
- Iceberg-1.18.2-1.0.44
- immersive_weathering-1.18.2-1.2.10
- inventorytotem_1.18.2-1.6
- ItemBorders-1.18.1-1.1.5
- journeymap-1.18.2-5.8.4-forge
- kelpfertilizer_1.18.2-1.8
- Kiwi-1.18.2-forge-6.1.0
- mcw-fences-1.0.5-mc1.18.2
- mcw-roofs-2.1.2-mc1.18.2
- mcw-windows-2.0.3-mc1.18.2
- memoryleakfix-1.18.2-0.3.2
- minecraft-comes-alive-7.3.0-forge
- musicdr-2.0.0
- mutil-1.18.2-4.5.0
- naturalist-forge-1.18.2-1.0.3
- NekosEnchantedBooks-1.18.2-1.7.0
- netherportalspread_1.18.2-5.7
- nohostilesaroundcampfire_1.18.2-3.9
- Patchouli-1.18.2-67
- perpetual_durability-1.18.2-1.1.0
- portality-1.18.2-3.3.4
- potionbundles-1.18.1-1.3
- realisticbees_1.18.2-2.5
- RoughlyEnoughItems-8.2.471
- rubidium_extras-1.18.2_v1.3.1
- rubidium-0.5.2a
- sapience-1.18.2-1.0.0
- scaffoldingdropsnearby_1.18.2-1.6
- selene-1.18.2-1.17.1
- SereneSeasons-1.18.2-7.0.0.15
- sleepwarp-1.1.1+1.18.2
- SnowRealMagic-1.18.2-forge-4.1.0
- structure_gel-1.18.2-2.3.0
- supplementaries-1.18.2-1.4.1
- TerraBlender-forge-1.18.2-1.1.0.101
- Terralith_v2.2.1a
- tetra-1.18.2-4.9.2
- titanium-1.18.2-3.5.5-37
- ToughAsNails-1.18.2-7.0.0.73
- Towns-and-Towers-1.18.2-1.6.0
- villagebellrecipe_1.18.2-1.4
- wooltweaks_1.18.2-1.5
- zaynens_leather_from_rotten_flesh_mod_1.18.2
- zetter-0.14.0-1.18.2-beta
- zombiehorsespawn_1.18.2-3.1

### Log

```shell
Can't provide log without execeding character limit.
```

**Comment by IchHabeHunger54** [comment]
The book being empty is normal as of now, the text is not yet implemented.
What is weird, however, is that the items don't appear above the lectern. Could you try reproducing the issue in a new instance with only Ars Magica (and Patchouli) installed?

**Comment by Sweed-Le-Deedly** [comment]
Okay, knowing that it's definately a compability issue. I took out all mods and ran a new save, and it worked!

![Screenshot 2022-06-05 112525](https://user-images.githubusercontent.com/69220229/172059174-d533d2a6-5e28-4718-a986-707eb04b5cf6.png)

After messing with taking out some mods, I found that the problem causing the issue was Rubidium, which I had been using for a dynamic lighting mod. I'm thinking this mod caused the items to not render to due to it reprograming the way Minecraft renders objects in order to speed up frame rate.

![Screenshot 2022-06-05 113147](https://user-images.githubusercontent.com/69220229/172059308-cf904be1-74d2-4934-a11e-4a3a2c07bfb6.png)
_Original mod testing world without Rubidium_

So, as this mod currently stands, it is incompatible with Rubidium in reguards to way the Crafting Altar renders items. Since it's rendering based, it may be similarly incompatible with other mods that change Minecraft's rendering. But, most of the ones I've seen are for Fabric and not Forge, so such instances may be rare. Maybe OptiFine, if I were to guess?

Anyways, thanks for your time!
